viewing_party/party.py

```diff
@@ -1,23 +1,152 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    if not title or not genre or not rating:
+        return None
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+    return new_movie

```

Comment:
This is a matter of personal preference and what reads easiest for you, but since the variable new_movie is only used in the return statement, we could factor it out by directly returning the new dictionary. Something like this:
```
return {
        "title": title,
        "genre": genre,
        "rating": rating,
}
```